reports/012/reproreview_012.Rmd

```diff
@@ -0,0 +1,131 @@
+                     svg{M108.9,79.1h41.6c39.6,0,57,28.3,57,53.6c0,27.5-21.5,53.6-56.8,53.6h-41.8V79.1z     M124.3,172.4h24.5c34.9,0,42.9-26.5,42.9-39.7c0-21.5-13.7-39.7-43.7-39.7h-23.7V172.4z}
+                     svg{M88.7,56.8c0,5.5-4.5,10.1-10.1,10.1c-5.6,0-10.1-4.6-10.1-10.1c0-5.6,4.5-10.1,10.1-10.1C84.2,46.7,88.7,51.3,88.7,56.8z};
+      }
+    }
+    \newcommand\orcid[1]{\href{https://orcid.org/#1}{\raisebox{0.15 em}{\mbox{\scalerel*{
+    \begin{tikzpicture}[yscale=-1, transform shape]
+    \pic{orcidlogo};
+    \end{tikzpicture}
+    }{|}}}}}
+    \definecolor{agileblue}{RGB}{0,77,155}
+urlcolor: agileblue
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+def.chunk.hook  <- knitr::knit_hooks$get("chunk")
+knitr::knit_hooks$set(chunk = function(x, options) {
+  x <- def.chunk.hook(x, options)
+  ifelse(options$size != "normalsize", paste0("\n \\", options$size,"\n\n", x, "\n\n \\normalsize"), x)
+})
+```
+
+```{r logo, eval=TRUE, echo=FALSE, message=FALSE, fig.align='center', out.width='0.3\\linewidth', fig.pos='H'}
+temp <- tempfile(fileext = ".pdf")
+download.file(url = "https://reproducible-agile.github.io/public/images/reproducible-AGILE-logo-square.pdf", destfile = temp)
+knitr::include_graphics(temp)
+```
+
+This report is part of the reproducibility review at the AGILE conference.
+For more information see [https://reproducible-agile.github.io/](https://reproducible-agile.github.io/).
+This document is published on OSF at [https://osf.io/wncsm/](https://osf.io/w7vph/).
+To cite the report use
+
+Koukouraki, E. (2022, June).  Reproducibility review of: Exploratory Analysis and Feature Selection for the Prediction of Nitrogen Dioxide.  [https://doi.org/10.17605/OSF.IO/WNCSM](https://doi.org/10.17605/OSF.IO/W7VPH)
+
+# Reviewed paper
+
+Iskandaryan, D., Di Sabatino, S., Ramos, F., and Trilles, S.: Exploratory Analysis and Feature Selection for the Prediction of Nitrogen Dioxide, AGILE GIScience Ser., 3, 6, [https://doi.org/10.5194/agile-giss-3-6-2022](https://doi.org/10.5194/agile-giss-3-6-2022)
+
+# Summary
+
+The paper evaluates the competence of selected features in the prediction of Nitrogen Dioxide with Machine Learning. For this reproduciblity review, the Figures and Tables of *Section 5 - Experiments and Results* were considered, while the Figures of *Section 3 - Exploratory Analysis* were not. The code of the corresponding analysis can be found in the following archived Github repository: [https://github.com/reproducible-agile/ExploratoryAnalysis_FeatureSelection](https://github.com/reproducible-agile/ExploratoryAnalysis_FeatureSelection). The data that is necessary to run the code can be found in the following Zenodo repository: [https://zenodo.org/record/6543074](https://zenodo.org/record/6543074). The reproduced results were in accordance with the ones reported in the paper, so the **reproduction** of the paper is considered **successful**.
```

Comment:
Add links to the original GitHub repository - our fork is a backup, but we want to point to the original source. Or put in both links once.

Reply:
Both links are now mentioned in the report.